cpu/efm32_common/periph/rtc.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (C) 2016-2017 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_efm32_common
+ * @ingroup     drivers_periph_rtc
+ * @{
+ *
+ * @file
+ * @brief       RTC peripheral driver implementation
+ *
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ * @}
+ */
+
+#include <time.h>
+
+#include "cpu.h"
+
+#include "periph_conf.h"
+#include "periph/rtc.h"
+
+#include "em_cmu.h"
+#include "em_rtc.h"
+
+#if defined(RTC_COUNT) && RTC_COUNT > 0
```

Comment:
I think these guards should be dropped now, as the RTC is only build if the feature is provided by the board/cpu, which again should only be the case if a valud RTC configuration is present...